File: app/src/main/java/co/otaoto/ViewActivity.java
package co.otaoto;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import butterknife.ButterKnife;
import butterknife.OnClick;

public class ViewActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_view);
        ButterKnife.bind(this);
    }

    @OnClick(R.id.view_create_button)
    void onCreateClick() {
        startActivity(new Intent(this, MainActivity.class));
        finish();
    }
}

File: app/src/main/java/co/otaoto/ViewActivity.kt
package co.otaoto

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import butterknife.ButterKnife
import butterknife.OnClick

class ViewActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_view)
        ButterKnife.bind(this)
    }

    @OnClick(R.id.view_create_button)
    internal fun onCreateClick() {
        startActivity(Intent(this, MainActivity::class.java))
        finish()
    }
}
